storage/src/backend/connection.rs

```diff
@@ -363,7 +348,50 @@ impl Connection {
                 });
             }
         }
-        // TODO: check mirrors' health
+        // Check mirrors' health
+        for mirror in connection.mirrors.iter() {
+            let conn = connection.clone();
+            let connect_timeout = config.connect_timeout;
+            // Spawn thread to update the health status of mirror server
+            let mirror_cloned = mirror.clone();
+            thread::spawn(move || {
+                let mirror_health_url = format!("{}/v2", mirror_cloned.config.host);
```

Comment:
Why use this URL to check healthy?

Reply:
Because we just check whether the mirror server is alive. If the server response is 401, we also think it is alive.

Maybe we should discuss it again.

Reply:
How about using the ping API provided by d7y dfdaemon: `127.0.0.1:40901/server/ping`, of course, the ping endpoint URL should support configuring by users.

Reply:
It is more reasonable.
How to deal with mirror servers without `ping` API? If users do not provide `ping` API, use mirror_server/v2 or disable it?

Reply:
It's ok to use `/v2` as default.

Reply:
Done. PTAL